packages/protocol/package.json

```diff
@@ -20,7 +20,8 @@
     "gas": "yarn run test --gas",
     "build:sol": "ts-node ./scripts/build.ts --solidity ${BUILD_DIR:-./build}",
     "build": "yarn build:sol && yarn build:ts",
-    "migrate": "./scripts/bash/migrate.sh",
+    "publish": "ts-node ./scripts/publish.ts",
```

Comment:
Maybe we could update the `verify` command below instead? Did you check what it does?

Reply:
I'd rather these remain as separate scripts. `verify` is a static local check, very different concerns imo.

Reply:
Looking at the [code](https://github.com/celo-org/celo-monorepo/blob/master/packages/protocol/scripts/bash/verify_contracts.sh), it just attempts to verify it on Blockscout, which is being replaced by being verified on Sourcify directly. I'd say it's the same concern, no?

Reply:
Oh right, never mind, was thinking of a different script. Agreed on moving this to the `verify` script.